pimd/pim_msdp_packet.c

```diff
@@ -634,16 +634,11 @@ static int pim_msdp_read_packet(struct pim_msdp_peer *mp)
 	nbytes = stream_read_try(mp->ibuf, mp->fd, readsize);
 	new_endp = stream_get_endp(mp->ibuf);
 	if (nbytes < 0) {
-		if (PIM_DEBUG_MSDP_INTERNAL) {
-			zlog_debug("MSDP peer %s read failed %d", mp->key_str,
-				   nbytes);
-		}
+		zlog_warn("MSDP peer %s read failed %d", mp->key_str, nbytes);
```

Comment:
I'm not sure that this rises to a warn level.  non-blocking io can return the fact that we could not read.  This makes no sense to expose it to the end user.  It was appropriately debug guarded imo.